media-sound/murmur/murmur-9999.ebuild

```diff
@@ -93,36 +94,40 @@ src_prepare() {
 	# Adjust systemd service file to our config location #689208
 	sed "s@/etc/${PN}\.ini@/etc/${PN}/${PN}.ini@" \
 		-i scripts/${PN}.service || die
+
+	cmake_src_prepare
 }
 
 src_configure() {
 	myuse() {
 		[[ -n "${1}" ]] || die "myconf: No use flag given."
 		use ${1} || echo "no-${1}"
 	}
-	local conf_add=(
-		no-client
-		$(myuse dbus)
-		$(usex debug 'symbols debug' release)
-		$(myuse ice)
-		$(myuse pch)
-		$(usex zeroconf '' no-bonjour)
+	local mycmakeargs=(
+		"-Doverlay=OFF"
+		"-Dserver=ON"
+		"-Dclient=OFF"
+		"-Ddbus=$(usex dbus)"
+		"-Dg15=OFF"
+		"-Dice=$(usex ice)"
+		"-Dgrpc=OFF"
+		"-DBUILD_TESTING=$(usex test)"
+		"-Dzeroconf=$(usex zeroconf)"
 	)
 
-	eqmake5 main.pro -recursive \
-		CONFIG+="${conf_add[*]}"
+	CMAKE_BUILD_TYPE=$(usex debug Debug Release) \
```

Comment:
You're ignoring CFLAGS from make.conf, use the default build type instead.

Reply:
Thanks for the review! I've removed the line after having checked other ebuilds using the `cmake` ECLASS

Reply:
But if `debug` USE flag is not used now, remove it from IUSE

Reply:
Hm, not so sure about that one... keepassxc also uses the `cmake` eclass, has debug in IUSE but doesn't actually use it inside the ebuild:
https://github.com/gentoo/gentoo/blob/64fe952caa63f4916a5e2f704ebd5306ff00198f/app-admin/keepassxc/keepassxc-2.6.2.ebuild#L27

Reply:
Hehe, nice one. I thought maybe the cmake eclass would use that info somehow. I've removed the USE flag from my proposed murmur-update now

Reply:
Usually if a eclass makes use of a USE flag, it adds it to IUSE itself